chart/openfaas/values.yaml

```diff
@@ -171,8 +171,9 @@ nats:
   image: nats-streaming:0.17.0
   enableMonitoring: false
   metrics:
```

Comment:
Could you add this as explicitly off in the ARM values files?

Something like this, plus your comment should do the job?
```yaml
nats:
  enableMonitoring: false
```